config.example.edn

```diff
@@ -43,15 +64,22 @@
                                      [:link {:rel "mask-icon" :color "#5bbad5" :href "/favicon/safari-pinned-tab.svg"}]
                                      [:link {:rel "shortcut icon" :href "/favicon/favicon.ico"}]
                                      [:link {:rel "manifest" :href "/favicon/site.webmanifest"}]]
- :triangulum.views/gtag-id          "G-BDLCXBE457"          ; update it
+ :triangulum.views/gtag-id          "G-" ; REVIEW
  :triangulum.views/static-css-files ["/css/comimo_global.css"] ; in resources/public/
  :triangulum.views/static-js-files  ["/js/jquery-3.4.1.min.js"
-                                     "/js/tabulator-4.5.3.min.js"
-                                     "https://www.gstatic.com/charts/loader.js"]
+                                     "/js/google-charts-loader.js"
```

Comment:
js/google-charts-loader.js is not found in the `resources` folder. Either commit the file or use link directly: "https://www.gstatic.com/charts/loader.js"